backend/.env.example

```
@@ -0,0 +1,3 @@
+NODE_ENV=development
+DATABASE_URL=./database.db
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The DATABASE_URL key should go before the NODE_ENV key

(UnorderedKey)